man/machinery-inspect-docker.1.md

```diff
@@ -0,0 +1,111 @@
+This approach ensures that no containers and images are affected by the inspection.
+
+The system data is structured into scopes, controlled by the
+`--scope` option.
+
+**Note**:
+Machinery will always inspect all specified scopes, and skip scopes which
+trigger errors.
+
+
+### ARGUMENTS
+
+  * `IMAGENAME / IMAGEID` (required):
+    The name or id of the image to be inspected. The name / id will also be
+    used as the name of the stored system description unless another name is
+    provided with the `--name` option.
+
+
+### OPTIONS
+
+  * `-d`, `--docker` (optional):
+    Defines the type of the image to be inspected as docker image.
+
+  * `-n NAME`, `--name=NAME` (optional):
+    Store the system description under the specified name.
+
+  * `-s SCOPE`, `--scope=SCOPE` (optional):
+    Inspect system for specified scope.
+    See the [Scope section](#Scopes) for more information.
+
+  * `-e SCOPE`, `--exclude-scope=EXCLUDE-SCOPE` (optional):
+    Inspect system for all scopes except the specified scope.
+    See the [Scope section](#Scopes) for more information.
+
+  * `-x`, `--extract-files` (optional):
+    Extract changed configuration and unmanaged files from the inspected system.
+    Shortcut for the combination of `--extract-changed-config-files`,
+    `--extract-unmanaged-files`, and `--extract-changed-managed-files`
+
+  * `--extract-changed-config-files` (optional):
+    Extract changed configuration files from the inspected system.
+
+  * `--extract-unmanaged-files` (optional):
+    Extract unmanaged files from the inspected system.
+
+  * `--extract-changed-managed-files` (optional):
+    Extract changed managed files from inspected system.
+
+  * `--skip-files` (optional):
+    Do not consider given files or directories during inspection. Either provide
+    one file or directory name or a list of names separated by commas. You can
+    also point to a file which contains a list of files to filter (one per line)
+    by adding an '@' before the path, e.g.
+
+      $ `machinery` inspect-container --docker --skip-files=@/path/to/filter_file myhost
+
+    If a filename contains a comma it needs to be escaped, e.g.
+
+      $ `machinery` inspect --skip-files=/file\\,with_comma myhost
+
+    **Note**: File or directory names are not expanded, e.g. '../path' is taken
+      literally and not expanded.
+
+  * `--verbose` (optional):
+    Display the filters which are used during inspection.
+
+
+### PREREQUISITES
+
+  * Inspecting a container requires an image of the defined container-type specified by the name or id.
+
+  * The system to be inspected needs to have the following commands:
+
+    * `rpm`
+    * `zypper` or `yum`
+    * `rsync`
+    * `chkconfig`
+    * `cat`
+    * `sed`
+    * `find`
+
+### EXAMPLES
+
+  * Inspect docker-container `myhost` / `076f46c1bef1` and save system description under name
```

Comment:
I'd make it two sections, one for myhost and one for 076f46c1bef1, because the `/` is confusing as it reads like a part of the command line.